src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -800,6 +800,11 @@ private static UserAuthenticationModel GetUserFromToken(JwtSecurityToken jwtSecu
                 }
             }
 
+            if (userAuthenticationModel.AuthenticationMethod.Equals(AuthenticationMethod.NotDefined))
```

Comment:
For some reason I don't like the use of `Equals` unless it is a string. I would have used the `==` operator.